gslab_fill/tablefill.py

```diff
@@ -77,39 +77,73 @@ def parse_data(data):
     return tables    
     
 
-def insert_tables(args,tables):
+def insert_tables(args, tables):
+    if re.search('\.tex', args['template']):
+        lyx_text = insert_tables_latex(args, tables)
+    else:

```

Comment:
I would prefer to splitting the function to `insert_tables_lyx` and `insert_tables_latex` and call the corresponding function based on `re.search('\.tex', args['template'])` or `re.search('\.lyx', args['template'])`